packages/core/src/components/ChoiceList/Choice.test.jsx

```diff
@@ -31,200 +41,125 @@ describe('Choice', () => {
   });
 
   it('is not checked', () => {
-    const props = {
```

Comment:
You can ignore these changes. This is just switching over to use the DRY'er `shallowRender` method. Skip down to the next comment where I point out what's actually new.